NIFI-11004 Add documentation for OIDC groups claim property (#6802)
apache/nifi: nifi-docs/src/main/asciidoc/administration-guide.adoc

|`nifi.security.user.oidc.additional.scopes` | Comma separated scopes that are sent to OpenId Connect Provider in addition to `openid` and `email`.
|`nifi.security.user.oidc.claim.identifying.user` | Claim that identifies the user to be logged in; default is `email`. May need to be requested via the `nifi.security.user.oidc.additional.scopes` before usage.
|`nifi.security.user.oidc.fallback.claims.identifying.user` | Comma separated possible fallback claims used to identify the user in case `nifi.security.user.oidc.claim.identifying.user` claim is not present for the login user.
|`nifi.security.user.oidc.claim.groups` | Name of the ID token claim that contains an array of group names of which the

> user is a member. Application groups must be supplied from a User Group Provider with matching names in order for the
> authorization process to use ID token claim groups. The default value is `groups`.

Reviewer: ```suggestion
user is a member. Application groups must be supplied by a User Group Provider with matching names for the
authorization process to use ID token claim groups. The default value is `groups`.
```
